src/pyscript/_generator.py

```diff
@@ -50,6 +51,10 @@ def create_project(
     app_dir.mkdir()
     manifest_file = app_dir / config["project_config_filename"]
     with manifest_file.open("w", encoding="utf-8") as fp:
-        json.dump(context, fp)
+        if str(manifest_file).endswith(".json"):
```

Comment:
Just wondering, what do you think about using fp.name instead of converting manifest_file to a string?

I guess manifest_file is more readable tho (unless we change the `fp` to something haha)

Reply:
There is a method on Path, [`.suffix`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix), that you might consider using instead of using string operations.